_data/sites/idris-maps.json

```diff
@@ -0,0 +1,7 @@
+{
+	"url": "https://idris-maps.com/",
+	"name": "idris-maps",
+	"description": "Anders\'s personal blog",
```

Comment:
I think it’s failing here, that `\` isn’t necessary

> /opt/build/repo/_data/sites/idris-maps.json: Unexpected token ' in JSON at position 84  at JSON.parse (<anonymous>)

Reply:
Sorry. Fixed.